packages/mongodb-redact/src/index.ts

```diff
@@ -1,26 +1,26 @@
-var _ = require('lodash');
-var regexes = require('./regexes');
+import _ from 'lodash';
+import { regexes } from './regexes';
 
-function redact(message) {
+export function redact<T>(message: T): T {
   if (_.isPlainObject(message)) {
     // recursively walk through all values of an object
-    return _.mapValues(message, redact);
+    return _.mapValues(message as any, redact);
   }
   if (_.isArray(message)) {
     // walk through array and redact each value
-    return _.map(message, redact);
+    return _.map(message, redact) as T;
   }
-  if (!_.isString(message)) {
+  if (typeof message !== 'string') {
```

Comment:
The lodash type check here also allows for an object string type: 
https://github.com/lodash/lodash/blob/ddfd9b11a0126db2302cb70ec9973b66baec0975/lodash.js#L12135
Is that something we should care about? I'm thinking fine as is, but I figured it's worth bringing up.
```
assert.strictEqual(_.isString(Object('a')), true);
```

Reply:
That's honestly pretty surprising to me... I guess technically `.replace()` would still work even on those, but tbh, I'd assume that the only reason this was using `_.isString()` was that whoever wrote this just used lodash for everything and didn't think much of this?

Reply:
Yeah I'd prefer we don't use lodash at all, but I think that's not something for now. All good!